docs/sources/operators-guide/configure/configure-helm-ha-deduplication-consul/index.md

```diff
@@ -0,0 +1,133 @@
+too to make Grafana Mimir and Prometheus/Grafana Agent config matches with each other, otherwise HA deduplication
+wouldn't work. `cluster` label value must be same across replica belong to the same cluster. `__replica__` must be
+unique across different replica in the cluster.
+
+```yaml
+global:
+  scrape_interval: 1m
+  scrape_timeout: 10s
+  evaluation_interval: 1m
+  external_labels:
+    __replica__: replica-1
+    cluster: my-prometheus
+```
+
+Reload or restart Prometheus or Grafana Agent after updating the configuration.
+
+## Install Consul using Helm
+
+To get and install Consul on Kubernetes, use the [Consul helm chart](https://github.com/hashicorp/consul-k8s/tree/main/charts/consul).
+Make a note of the Consul endpoint, because you will need it later to configure Mimir.
+
+## Configure Mimir high availability
+
+You can configure Mimir HA deduplication globally or at the tenant level.
+
+### Globally configure HA deduplication
+
+Before you begin, make sure that Mimir was installed using the Helm chart.
+
+1. Add the following configuration to your `custom.yaml` file:
+
+```yaml
+mimir:
+  structuredConfig:
+    limits:
+      accept_ha_samples: true
+      # The following two configurations must match those of external_labels in Prometheus
+      # The config values below are the default and can be removed if you don't want to override to a new value
+      ha_cluster_label: cluster
+      ha_replica_label: __replica__
+    distributor:
+      ha_tracker:
+        enable_ha_tracker: true
+        kvstore:
+          store: consul
+          consul:
+            host: <consul-endpoint> # example: http://consul.consul.svc.cluster.local:8500
+```
+
+2. Upgrade the Mimir's helm release using the following command:
+
+```bash
+ helm -n <mimir-namespace> upgrade --install mimir grafana/mimir-distributed -f custom.yaml
+```
+
+### Configure HA deduplication per tenant
+
+1. Add the following configuration to the `custom.yaml` file:
+
+```yaml
+runtimeConfig:
+  overrides:
+    <tenant-id>: # put real tenant ID here
+      accept_ha_samples: true
+      ha_cluster_label: cluster
+      ha_replica_label: __replica__
+```
+
+2. Upgrade the Mimir's helm release using the following command:
+
+```bash
+ helm -n <mimir-namespace> upgrade --install mimir grafana/mimir-distributed -f custom.yaml
+```
+
+## Verifying deduplication
+
+After Consul, Prometheus and Mimir running we can verify deduplication in several way. 
+
+### Distributor metrics
+
+Mimir [distributor]{{< relref "../../architecture/components/distribuotr.md" >}} exposes some metrics related 
+to HA deduplication. The relevant metrics are those with `cortex_ha_tracker_` prefix. 
+
+### ha_tracker's page
+
+Port forward Mimir distributor service. The argument after port-forward must
+match your Mimir's distributor name.
+
+```bash
+kubectl port-forward service/mymimir-distributor 8080:8080
```

Comment:
```suggestion
kubectl -n <mimir-namespace> port-forward service/mimir-distributor 8080:8080
```
Also we could say that to get the distributor service name, you can use this command:
```
kubectl -n <mimir-namespace> get service | grep distributor
```